qlty-check/src/executor/invocation_result.rs

```diff
@@ -280,9 +280,18 @@ impl InvocationResult {
     fn handle_output_rewrite(&mut self) -> Result<()> {
         let mut formatted = vec![];
 
-        for workspace_entry in self.plan.workspace_entries.iter() {
-            let workspace_path = self.plan.workspace.root.join(&workspace_entry.path);
-            let staged_path = self.plan.target_root.join(&workspace_entry.path);
+        for workspace_entry in self.invocation.target_paths.iter() {
+            let prefixed_path = if let Some(prefix) = &self.plan.plugin.prefix {
+                PathBuf::from(prefix).join(workspace_entry)
+            } else {
+                PathBuf::from(&workspace_entry)
+            };
+
+            let workspace_path = self.plan.workspace.root.join(&prefixed_path);
+            let staged_path = self.plan.target_root.join(workspace_entry);
```

Comment:
Aside from the log lines, are we using both of these variables?

Reply:
Yes they are being used in existing code below.